spark3/src/main/java/org/apache/iceberg/spark/source/SparkStreamingWrite.java

```diff
@@ -40,7 +40,7 @@
 import org.slf4j.Logger;
 import org.slf4j.LoggerFactory;
 
-public class SparkStreamingWrite extends SparkBatchWrite implements StreamingWrite {
+class SparkStreamingWrite extends BaseBatchWrite implements StreamingWrite {
```

Comment:
In my changes for `RequiresDistributionAndOrdering`, this class went away and is replaced by an inner class. I think that pattern worked well. Maybe we could do that before this one to reduce the number of changes here.

Reply:
Let me see that change.